src/main/java/com/treasure/hunt/game/GameManager.java

```diff
@@ -384,4 +398,75 @@ public GameManager copy(Kryo kryo) {
         gameManager.setBindings();
         return gameManager;
     }
+
+    /**
+     * @param coordinate the point on the canvas, we want to get the closest {@link GeometryType} to.
+     * @param distance   the maximum distance to a potential {@link GeometryItem}.
+     * @return a sorted list, containing the nearest {@link GeometryItem}'s to {@code coordinate}, with a maximum distance of {@code distance}.
+     */
+    private List<Selectable> pickGeometryItem(Coordinate coordinate, double distance) {
+        List<Turn> visibleTurns = getVisibleTurns();
+        List<Selectable> selectables = new ArrayList<>();
```

Comment:
Why is this here and not in s separate Class?

Reply:
> AHHH one change-> one issue-> one PR
> 
> You know what happens if I/we/whoever dislikes one of the changes... You will sit there redoing them or trying to separate them

No problem, when the PR's are solved in the chronological order, which is important in this case. Had to fix things in the older PR's which are important to solve Problems in the newer PR's.